content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'scaleOrdinal()'
+Description: 'scaleOrdinal() function and examples'
+Subjects:
+  - 'Data visualization'
+  - 'Web design'
+Tags: 
+  - 'D3'
+CatalogContent:
+--- 
+
+**scaleOrdinal()** function is used to generate and return an ordinal scale with the supplied domain and range. The array is set to be an empty array, if neither the range nor the domain are specified.
+
+## Syntax
+
+It has two input parameters: domain and range:
+
+scaleOrdinal(domain, range)
+
+- The domain specifies the scale's maximum and minimum values.
+- Range. Values in the range corresponds to values in the domain.
+
+d3.scaleOrdinal([[domain, ]range]);
+
+## Example
+
+Below example shows inventions corresponding to associated year of invention. 
```

Comment:
This sentence should tell us what the `.scaleOrdinal()` is doing in the example below.

Reply:
Updated